guides/06_other-tutorials/developing-faster-with-reload-mode.md

```diff
@@ -54,6 +54,11 @@ The important part here is the line that says `Watching...` What's happening her
 
 As a small aside, this auto-reloading happens if you change your `app.py` source code or the Gradio source code. Meaning that this can be useful if you decide to [contribute to Gradio itself](https://github.com/gradio-app/gradio/blob/main/CONTRIBUTING.md) ✅
 
+⚠️ It is customary to configure the behavior of your gradio app via the `launch()` or `queue()` methods. For example, setting `auth`, or `show_error` in `launch()`. If this is the case, make sure that these methods are
```

Comment:
This only applies to `launch()`, right? You could have `queue()` outside of the block and everything should work fine I believe

Reply:
Actually no because the queue doesn't get configured